### PR TITLE
Feature/thematische subsidie

### PR DESCRIPTION
**Pull request by aatauil**
Adds a new subsidie formulier. It is identical to stadsvernieuwig conceptsubsidy with the only difference being the "aanvragen tot" date

To test:
- mock login
- go to subsidy and create thematische subsidie
(note: not all bestuurseenheden have this subsidy, use testMode if you dont see the subsidy)
- fill in save and load subsidy

### CODE REVIEW

File: config/subsidy-application-management/forms/urban-renewal-thematic/request/versions/20230427140454/form.ttl [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    # Hidden field required for all variations of URL or FILE
+    # input field which require validation.
+    # It makes sure there is a type attached to hasPart object.
+    # This enables correct validation in both front and backend.
+  ##########################################################
+  ext:hiddenAttachmentsF a form:Field ;
+    sh:order 4 ;
+    sh:name "Type RemoteDataObject or FileDataObject [hidden input]" ;
+    sh:path ( lblodSubsidie:urbanRenewalAttachments dct:hasPart rdf:type );
+    sh:group ext:applicationPg .
```
Comment (Windvis) [inline config/subsidy-application-management/forms/urban-renewal-thematic/request/versions/20230427140454/form.ttl:200]: Is this still needed if the field has no validation?

File: config/subsidy-application-management/forms/urban-renewal-thematic/request/versions/20230427140454/form.ttl [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    sh:name "Aanvraag" ;
+    sh:order 2 .
+
+  ##########################################################
+  #  field: application form
+  ##########################################################
+  ext:applicationFormF a form:Field ;
+    sh:order 1 ;
+    sh:name "Laad hier het ingevuld aanvraagformulier op." ;
+    form:help """Het aanvraagformulier vindt u terug op deze <a href="https://www.vlaanderen.be/stedenbeleid/stadsvernieuwing-in-vlaanderen" target="_blank">webpagina</a>.""" ;
```
Comment (Windvis) [inline config/subsidy-application-management/forms/urban-renewal-thematic/request/versions/20230427140454/form.ttl:153]: Nitpick, but it makes the link look nicer.
```suggestion
    form:help """Het aanvraagformulier vindt u terug op deze <a href="https://www.vlaanderen.be/stedenbeleid/stadsvernieuwing-in-vlaanderen" class="au-c-link" target="_blank">webpagina</a>.""" ;
```

**Review by Windvis — COMMENTED** [review]
Seems to work as expected when I used the ?testMode=true flag.

What organizations should have access to this subsidy?

File: config/subsidy-application-management/forms/urban-renewal-thematic/request/versions/20230427140454/form.ttl (hunk 1 above)
Comment (aatauil) [reply config/subsidy-application-management/forms/urban-renewal-thematic/request/versions/20230427140454/form.ttl:200]: If I remember correctly I am pretty sure yes, has something to do with how we model it in the backend. we have 2 links for a file, a virtual one and a physical one

File: config/migrations/2023/20230426112611-thematische-subsidie/20230426112611-add-thematische-subsidie.sparql [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    <http://lblod.data.info/id/subsidie-application-flow-steps/5f3af9d1-692a-4518-ab45-2d22b43f31f1> a lblodSubsidie:ApplicationStep;
+      mu:uuid "5f3af9d1-692a-4518-ab45-2d22b43f31f1";
+      qb:order 0;
+      dct:references <http://data.lblod.info/id/subsidy-procedural-steps/903a4cf5-1b57-44ed-8c00-a0c71a0d3970>;
+      dct:isPartOf  <http://lblod.data.info/id/subsidie-application-flows/0c7e217b-fec9-4a56-9024-35c6db006fa3>;
+      dct:source <config://forms/urban-renewal-thematic/request/versions/20230427140454/form.ttl>.
+
+    <http://data.lblod.info/id/periodes/ecb5b813-16c9-44aa-9b49-262a84ff4301> a <http://data.europa.eu/m8g/PeriodOfTime>;
+      mu:uuid "ecb5b813-16c9-44aa-9b49-262a84ff4301";
+      <http://data.europa.eu/m8g/startTime> "2023-04-24T03:00:00Z"^^<http://www.w3.org/2001/XMLSchema#dateTime>;
```
[the remark this answers is not in the document]
Comment (aatauil) [reply config/migrations/2023/20230426112611-thematische-subsidie/20230426112611-add-thematische-subsidie.sparql:52]: They asked for a copy of the existing stadsvernieuwing subsidies so I assume yes.

**Comment by aatauil** [comment]
@Windvis added missig criterion and the link class addition, thanks for the review :pray:

**Pull request merged**